src/cclib/io/filewriter.py

```diff
@@ -89,6 +92,13 @@ def _make_bond_connectivity_from_openbabel(self, obmol):
                                         obbond.GetBondOrder()))
         return bond_connectivities
 
+    def _check_required_attributes(self, required):
+        """Check if required attributes are present in ccdata"""
+        missing = [x for x in required if not hasattr(self.ccdata, x)]
+        if len(missing) > 0:
+            missing = ' '.join(missing)
+            raise MissingAttributeError('Could not parse required outputs to write molden file: ' + missing)
```

Comment:
Removed. New error message would look something like 
`Could not parse required outputs to write file: gbasis, natom`